doc/whats_new/v0.22.rst

```diff
@@ -112,6 +112,13 @@ Changelog
   need to store the entire dense matrix at once.
   :pr:`13960` by :user:`Scott Gigante <scottgigante>`.
 
+:mod:`sklearn.dummy`
+............................
+
+- |Fix| :class:`dummy.DummyClassifiers()` now handles checking the existence
+  of the provided constant in multiouput cases.
+  :pr:`14908` by `Martina G. Vilas`_.
```

Comment:
You would need to add user username in one of the file. Using `:user:` will be easier.

```suggestion
  :pr:`14908` by :user:`Martina G. Vilas <martinagvilas>`.
```